at_client/lib/src/service/connectivity_listener.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:async';
+
+//import 'package:data_connection_checker/data_connection_checker.dart';
+import 'package:internet_connection_checker/internet_connection_checker.dart';
+class ConnectivityListener {
```

Comment:
@murali-shris @kalluriramkumar  sample code of usage in the dart docs just like the notification service

Reply:
done